cwltool/command_line_tool.py

```diff
@@ -169,9 +169,9 @@ def revmap_file(builder, outdir, f):
 
         if revmap_f and not builder.pathmapper.mapper(revmap_f[0]).type.startswith("Writable"):
             f["location"] = revmap_f[1]
-        elif uripath == outdir or uripath.startswith(outdir+os.sep):
+        elif uripath == outdir or uripath.startswith(outdir+os.sep) or uripath.startswith(outdir+'/'):
```

Comment:
Does this fix a bug? I'd like to see a unit test that this change fixes.

I'm confused as to why it is `os.sep` and not always `'/'`

Reply:
It was failing on Appveyor.  For some reason it was mixing `/` and `\` for path separators, and os.sep is `\` on Windows.  But then I added windows_requires_docker which causes that code to be skipped completely.  So I think there may be a Windows specific bug, and this is a brute force workaround.

Reply:
So, I believe this fixes a legitimate bug, and that mostly things don't fail on appveyor because we skip those tests, not because it was actually working before.  However I don't have access to a windows install to probe it thoroughly.  So I can leave it in as a basically harmless fix or take it out and know what we probably have a windows specific bug (that won't get fixed until someone else digs into it more deeply.)